spotbugs-exclude.xml

```diff
@@ -3,6 +3,8 @@
   <Match>
     <Or>
       <Bug pattern="DM_STRING_CTOR"/>
+      <Bug pattern="EI_EXPOSE_REP"/>
+      <Bug pattern="EI_EXPOSE_REP2"/>
```

Comment:
Exclude `EI_EXPOSE_REP` and `EI_EXPOSE_REP2`. We can address them in dedicated PR.
```
[ERROR] Medium: new com.google.cloud.tools.maven.cloudsdk.CloudSdkAppEngineFactory(CloudSdkMojo) may expose internal representation by storing an externally mutable object into CloudSdkAppEngineFactory.mojo [com.google.cloud.tools.maven.cloudsdk.CloudSdkAppEngineFactory] At CloudSdkAppEngineFactory.java:[line 56] EI_EXPOSE_REP2
[ERROR] Medium: com.google.cloud.tools.maven.cloudsdk.CloudSdkMojo.getMavenProject() may expose internal representation by returning CloudSdkMojo.mavenProject [com.google.cloud.tools.maven.cloudsdk.CloudSdkMojo] At CloudSdkMojo.java:[line 125] EI_EXPOSE_REP
[ERROR] Medium: com.google.cloud.tools.maven.cloudsdk.CloudSdkMojo.getMavenSession() may expose internal representation by returning CloudSdkMojo.mavenSession [com.google.cloud.tools.maven.cloudsdk.CloudSdkMojo] At CloudSdkMojo.java:[line 129] EI_EXPOSE_REP
[ERROR] Medium: com.google.cloud.tools.maven.run.AbstractRunMojo.getAdditionalArguments() may expose internal representation by returning AbstractRunMojo.additionalArguments [com.google.cloud.tools.maven.run.AbstractRunMojo] At AbstractRunMojo.java:[line 119] EI_EXPOSE_REP
[ERROR] Medium: com.google.cloud.tools.maven.run.AbstractRunMojo.getEnvironment() may expose internal representation by returning AbstractRunMojo.environment [com.google.cloud.tools.maven.run.AbstractRunMojo] At AbstractRunMojo.java:[line 115] EI_EXPOSE_REP
[ERROR] Medium: com.google.cloud.tools.maven.run.AbstractRunMojo.getJvmFlags() may expose internal representation by returning AbstractRunMojo.jvmFlags [com.google.cloud.tools.maven.run.AbstractRunMojo] At AbstractRunMojo.java:[line 103] EI_EXPOSE_REP
```

Reply:
I also feel these are inevitable for these getters.